plugins/CoreAdminHome/Commands/ConfigDelete.php

```diff
@@ -0,0 +1,319 @@
+        );
+        $this->addOption('section', 's', InputOption::VALUE_REQUIRED, 'The section the INI config setting belongs to.');
+        $this->addOption('key', 'k', InputOption::VALUE_REQUIRED, 'The name of the INI config setting.');
+        $this->addOption('value', 'i', InputOption::VALUE_REQUIRED, 'For arrays, specify the array value to be deleted.');
+
+        $this->setHelp("This command can be used to delete a INI config setting.
+
+You can delete config values per the two sections below, where:
+- [Section] is the name of the section, e.g. database or General.
+- [config_setting_name] the name of the setting, e.g. username.
+- [array_value] For arrays, the specific array value to delete.
+
+(1) By settings options --section=[Section] and --key=[config_setting_name], and optionally --value=[array_value].  Examples:
+#Delete this setting.
+$ ./console %command.name% --section=database --key=username
+#Delete one value in an array:
+$ ./console %command.name% --section=database --key=username --value=DevicesDetection
+
+OR
+
+(2) By using a command argument in the format [Section].[config_setting_name].[array_value]. Examples:
+#Delete this setting.
+$ ./console %command.name% 'database.username'
+#Delete one value in an array:
+$ ./console %command.name% 'PluginsInstalled.PluginsInstalled.DevicesDetection'
+
+NOTES:
+- Settings may still appear to exist if they are set in global.ini.php or elsewhere.
+- Section names, key names, and array values are all case-sensitive; so e.g. --section=Database fails but --section=database works.  Look in config.ini.php and global.ini.php for the proper case.
+- If no matching section/setting is found, the string \"" . self::MSG_NOTHING_FOUND . "\" shows.
+- For safety, this tool cannot be used to delete a whole section of settings or an array of values in a single command.
+");
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        //Optionally could set this at runtime with: $output->getVerbosity() >= OutputInterface::VERBOSITY_VERBOSE.
+        $debug = false;
+
+        // Gather options, then discard ones that are empty so we do not need to check for empty later.
+        $options = array_filter([
+            'section' => $input->getOption('section'),
+            'key' => $input->getOption('key'),
+            'value' => $input->getOption('value'),
+        ]);
+        $debug && fwrite(STDERR, PHP_EOL . __FUNCTION__ . '::Started with $options=' . (empty($options) ? '' : serialize($options)));
+
+        $argument = trim($input->getArgument('argument'));
+        $debug && fwrite(STDERR, PHP_EOL . __FUNCTION__ . '::Started with $argument=' . (empty($argument) ? '' : serialize($argument)));
+
+        // Sanity check inputs.
+        switch (true) {
+            case empty($argument) && empty($options):
+                throw new \InvalidArgumentException('You must set either an argument or set options --section and optional --key');
```

Comment:
Guess it should be this:
```suggestion
                throw new \InvalidArgumentException('You must set either an argument or set options --section, --key and optional --value');
```